qa/pom.xml

```diff
@@ -27,7 +27,6 @@
   <properties>
     <version.com.sun.messaging.mq.fscontext>4.6-b01</version.com.sun.messaging.mq.fscontext>
     <version.jboss.profiler.jvmti>1.0.0.CR5</version.jboss.profiler.jvmti>
-    <version.log4j-core>2.19.0</version.log4j-core>
```

Comment:
@jmfinelli is this line still needed please?

Reply:
"needed" as in "needed to be removed" : )

Reply:
I think that property can be removed since it is already present in the root pom.